packages/gmap-vue/src/components/heatmap-layer.js

```diff
@@ -0,0 +1,27 @@
+import mapElementFactory from '../factories/map-element'
+
+const props = {
+    options: {
+        type: Object,
+        twoWay: false,
+        default: () => {},
+    },
+    data: {
+        type: Array,
+        twoWay: true        
+    },
+};
+
+const events = [];
+
+/**
+ * @class Heatmap Layer
+ *
+ * Heatmap Layer class
+ */
+export default mapElementFactory({
+    mappedProps: props,
+    events: events,

```

Comment:
```suggestion
    events,
```
Picky: In order to use ES6 standards 👍🏽